submit/submit_online.bats

```diff
@@ -23,7 +23,7 @@ setup() {
 @test "hello problem id and name are in help output" {
     run ./submit --help
     assert_success
-    assert_regex "hello *- *Hello World"
+    assert_regex "A *- *Hello World"
```

Comment:
Oooohhh do we now also have 1 letter labels for the problems? I like it

Reply:
It also means we're not testing multi-character labels anymore...

Reply:
Which we should maybe do separately. On current main we are not testing single character labels…

Reply:
Yeah, it's defined in the `problems.yaml`. It looks much nicer and is the standard way of how DOMjudge is used.

But I agree we lose some test coverage here and probably want to add it back in a follow-up PR.